google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerImpl.java

```diff
@@ -218,9 +220,20 @@ public DatabaseClient getDatabaseClient(DatabaseId db) {
                 LabelValue.create(db.getDatabase()),
                 LabelValue.create(db.getInstanceId().getName()),
                 LabelValue.create(GaxProperties.getLibraryVersion(getOptions().getClass())));
+
+        AttributesBuilder attributesBuilder = Attributes.builder();
+        attributesBuilder.put("client_id", clientId);
+        attributesBuilder.put("database", db.getDatabase());
+        attributesBuilder.put("instance_id", db.getInstanceId().getName());
```

Comment:
This seems a bit inconsistent at first glance:
1. We add `database` with the ID of the database.
2. We add `instance_id` with the name of the instance.

Normally, we use this terminology:
* `projects/my-project/instance/my-instance`: This is an instance name. The instance ID is `my-instance`

Is this intentional because other Google Cloud products do it this way? Or would it maybe make more sense to either add `project_id`, `instance_id` and `database_id` all as separate attributes?

Reply:
These attributes names and values are kept same as what we had with OpenCensus. As these attributes might be used by customers to build dashboards and alerts, they would have to do these changes. 

For now, they will only change lets say the source, read metrics data from opentelemetry instead of opencensus.

Reply:
ack